test/process_client.py

```diff
@@ -185,6 +185,43 @@ def get_layer(username, layername, headers=None, assert_status=True,):
         return r
 
 
+def get_map(username, mapname, headers=None, assert_status=True, ):
+    headers = headers or {}
+
+    with app.app_context():
+        r_url = url_for('rest_map.get', username=username, mapname=mapname)
+    r = requests.get(r_url, headers=headers)
+    if assert_status:
+        assert r.status_code == 200, r.text
+        return r.json()
+    else:
+        return r
+
+
+def ensure_layer(username,
+                 layername,
+                 headers=None,
+                 access_rights=None,
+                 ):
+    headers = headers or {}
+    r = get_layer(username, layername, headers=headers, assert_status=False)
```

Comment:
get_layer používá rest a volá všechny source, navrhuji použít něco rychlejšího

Reply:
REST GET Layers vs. layman.util.get_publication_info(context=access_rights_source)
100 calls
1.42 s vs. 0.15 s
I valuate more using REST at this moment.